From d44df8d5b93bb9a273e1ab4750e749353bba4134 Mon Sep 17 00:00:00 2001
From: Ran Wang <ran.wang_1@nxp.com>
Date: Thu, 9 Aug 2018 11:41:18 +0800
Subject: [PATCH 407/448] armv7:ls1021a: Correct addr of RCPM_IPPDEXPCR1

Otherwise Flex timer will not be able to wakeup system.

Signed-off-by: Ran Wang <ran.wang_1@nxp.com>
---
 arch/arm/boot/dts/ls1021a.dtsi |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/ls1021a.dtsi b/arch/arm/boot/dts/ls1021a.dtsi
index 2d7036b..84dcfc8 100644
--- a/arch/arm/boot/dts/ls1021a.dtsi
+++ b/arch/arm/boot/dts/ls1021a.dtsi
@@ -516,7 +516,7 @@
 		ftm0: ftm0@29d0000 {
 			compatible = "fsl,ls1021a-ftm";
 			reg = <0x0 0x29d0000 0x0 0x10000>,
-			      <0x0 0x1ee2140 0x0 0x4>;
+			      <0x0 0x1ee2144 0x0 0x4>;
 			reg-names = "ftm", "FlexTimer1";
 			interrupts = <GIC_SPI 118 IRQ_TYPE_LEVEL_HIGH>;
 			big-endian;
-- 
1.7.1

